src/Sarif.Viewer.VisualStudio/CommonDialogFactory.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sarif.Viewer
+{
+    public class CommonDialogFactory : ICommonDialogFactory
+    {
+        public IOpenFileDialog CreateOpenFileDialog()
+        {
+            return new RealOpenFileDialog();
```

Comment:
Note on naming: I call the dialog wrapper class `RealOpenFileDialog`, rather than just `OpenFileDialog`, because there's a class `OpenFileDialog` in the `Microsoft.Win32` namespace. OTOH, I call the factory class `CommonDialogFactory`, rather than `RealCommonDialogFactory`, because there's no existing class for the name to conflict with.